storey/drivers.py

```diff
@@ -352,6 +355,8 @@ def _convert_python_obj_to_expression_value(value):
         elif isinstance(value, bytes):
             return f"blob('{base64.b64encode(value).decode('ascii')}')"
         elif isinstance(value, datetime):
```

Comment:
FYI, this will catch pandas NaT, but not numpy NaT:
`isinstance(np.datetime64("NaT"), datetime.datetime)` is False

Reply:
Will be addressed separately, as @katyakats says there are more issues with numpy time.